src/coreclr/tools/aot/ILCompiler.ReadyToRun/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -721,13 +697,11 @@ public static PEHeaderBuilder Create(Characteristics imageCharacteristics, DllCh
                 sectionAlignment = fileAlignment;
             }
 
-            dllCharacteristics &= DllCharacteristics.AppContainer;
```

Comment:
is `AppContainer` not required?

Reply:
I think this flag does nothing on DLLs.  The C# compiler does not even have an option to set it on a DLL.

Reply:
but in this case it can also be an exe or since its marked with `IMAGE_FILE_DLL` it does not matter?

Reply:
We always use a host and never run a R2R binary directly.  In what scenario can it be an .exe?

Reply:
ah yes.